apps/chat/channels.py

```diff
@@ -618,9 +618,6 @@ def send_voice_to_user(self, synthetic_voice: SynthesizedAudio):
 
 
 class SureAdhereChannel(ChannelBase):
-    def initialize(self):
-        self.messaging_service = self.experiment_channel.messaging_provider.get_messaging_service()
```

Comment:
There is already a `messaging_service` property with the same definition on the parent class